packages/elements-example-vue/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
BTW: Why don't we use vue with typescript? Is this not recommended? If recommended, let's create a follow up story for that.

Reply:
I will create a follow up story :)